release/RELEASE.md

```diff
@@ -198,6 +204,10 @@ latest commit on master.
    ./release.sh changes stable previous
    ```
 
+1. **[STABLE]** Update the
+   [create-daml-app](https://github.com/digital-asset/create-daml-app) template
+   to use the new version, and add a tag for it.
```

Comment:
We should say that the tag needs to be `vX.XX.XX` (the new version number).

Reply:
If the previous step (create-daml-app being on the latest snapshot) was done, then this should just be a matter of changing version numbers, though in several places (daml.yaml and package.json at least). Just confirming we want this burden on the releaser @garyverhaegen-da @hurryabit ?

Reply:
I think to put that burden on the person promoting the release is fine for now. We plan to do this once a month and the person is most likely somebody who'll specifically be asked to make the release, including all such hassles.